calmixRT.csv

```diff
@@ -0,0 +1,18 @@
+sequence,irt
+SSAAPPPPPR,33.18
+GISNEGONASIK,61
+HVLTSIGEK,39.45
+DIPVPKPK,47.28
+LSSEAPALFQFDLK,68.48
+IGDYAGIK,49.46
+TASEFDSAIAQDK,56.58
+SAAGAFGPELSR,61.02
+ELGQSGVDTYLQTK,66.73
+GLILVGGYGTR,78.15
+GILFVGSGVSGGEEGAR,79.32
+SFANQPLEVVYSK,78.2
+LTILEELR,88.05
+NGFILDGFPR,92.87
+ELASGLSFPVGFK,96.19
+LSSEAPALFQFDLK,101.78
+GISNEGQNASIK,36.7 
```

Comment:
this is a data file pushed to the root directory of the repo, this is not usually a good practice